packages/ts-sdk-apm/README.md

```diff
@@ -0,0 +1,75 @@
+# @commercetools/ts-sdk-apm
+
+_commercetools Typescript SDK Application Performance Monitoring._
+
+This package is used to monitor the application performance of the Typescript SDK. It exposes a middleware creator method that accepts some options for monitoring and tracing the SDK performance.
+
+## Installation
+
+To use this package we need to install the package and add the newrelic configuration file.
+
+Using npm
+
+```bash
+$ npm install @commercetools/ts-sdk-apm
+```
+
```

Comment:
IMO probably it would be better to add all the technical information such as how to install the agent here instead of https://github.com/commercetools/commercetools-docs/pull/3184/files, just to be more consistent.
what do you also think @jenschude ?

Reply:
I would not add details about installing the agent to the docs, as it's a 3rd party tool we are not providing support for it.